prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -55,7 +55,27 @@ postsubmits:
         resources:
           requests:
             memory: "1Gi"
-
+  - name: post-test-infra-gencred-refresh-kubeconfig
+    cluster: test-infra-trusted
+    run_if_changed: '^(prow/oss/gencred-config/|prow/prowjobs/GoogleCloudPlatform/oss\-test\-infra/gcp\-oss\-test\-infra\-config\.yaml)'
```

Comment:
The `-`s don't need to be backslash escaped. I'm not sure what effect that will have.

Reply:
removed